src/lib/select/select.html

```diff
@@ -19,31 +19,31 @@
   cdk-connected-overlay
   cdkConnectedOverlayLockPosition
   cdkConnectedOverlayHasBackdrop
+  [cdkConnectedOverlayOpen]="true"
   cdkConnectedOverlayBackdropClass="cdk-overlay-transparent-backdrop"
   [cdkConnectedOverlayScrollStrategy]="_scrollStrategy"
   [cdkConnectedOverlayOrigin]="origin"
-  [cdkConnectedOverlayOpen]="panelOpen"
   [cdkConnectedOverlayPositions]="_positions"
   [cdkConnectedOverlayMinWidth]="_triggerRect?.width"
   [cdkConnectedOverlayOffsetY]="_offsetY"
+  [cdkConnectedOverlayDetachOnEscape]="false"
   (backdropClick)="close()"
   (attach)="_onAttached()"
   (detach)="close()">
-
   <div
     #panel
     class="mat-select-panel {{ _getPanelTheme() }}"
+    [class.cdk-visually-hidden]="!panelOpen"
     [ngClass]="panelClass"
-    [@transformPanel]="multiple ? 'showing-multiple' : 'showing'"
+    [@transformPanel]="panelOpen ? (multiple ? 'showing-multiple' : 'showing') : 'void'"
```

Comment:
Would be a good idea to move this into a function rather than having the nested ternary.